### PR TITLE
Web3.js Adapter for Wagmi: Documenation

### LINKED ISSUE #6746: Web3.js Adapter for Wagmi: Implementation
- [ ] Create Wagmi adapter snippets (like https://wagmi.sh/react/guides/ethers ) for web3.js:
		Client → Provider
		Connector Client → Signer

### PR DESCRIPTION
**Pull request by avkos**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).

Fixes #6746 #6751 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: docs/docs/guides/wagmi_usage/wagmi.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    if (transport.type === 'fallback') {
+        return new Web3(transport.transports[0].value.url)
+    }
+
+    return new Web3(transport.url)
+}
+
+/** Action to convert a viem Client to a web3.js Instance. */
+export function useWeb3js({chainId}: { chainId?: number } = {}) {
```
Comment (jdevcs) [inline docs/docs/guides/wagmi_usage/wagmi.md:32]: @avkos are these manually tested ?

File: docs/docs/guides/wagmi_usage/wagmi.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 1
+sidebar_label: 'Wagmi Web3js Adaptor'
+title: 'Wagmi Web3js Adaptor'
+---
+
+If you're using [Wagmi](https://wagmi.sh/react/getting-started#use-wagmi) and want to add web3.js, use this provider in your project:
+
+
+```typescript
```
Comment (jdevcs) [inline docs/docs/guides/wagmi_usage/wagmi.md:12]: and for https://github.com/web3/web3.js/issues/6746 there should be signer adapter doc as well

File: docs/docs/guides/wagmi_usage/wagmi.md (hunk 1 above)
Comment (avkos) [reply docs/docs/guides/wagmi_usage/wagmi.md:32]: yes. here is the [dapp example ](https://github.com/avkos/wagmi-web3js-example-app)

**Review by jdevcs — APPROVED** [review]

File: docs/docs/guides/wagmi_usage/wagmi.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 1
+sidebar_label: 'Wagmi Web3js Adaptor'
+title: 'Wagmi Web3js Adaptor'
+---
+
+If you're using [Wagmi](https://wagmi.sh/react/getting-started#use-wagmi) and want to add web3.js, use this provider in your project:
```
Comment (jdevcs) [inline docs/docs/guides/wagmi_usage/wagmi.md]: lgtm, just add some more description for this part that these are adapter snippets

File: docs/docs/guides/wagmi_usage/wagmi.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 1
+sidebar_label: 'Wagmi Web3js Adaptor'
```
Comment (jdevcs) [inline docs/docs/guides/wagmi_usage/wagmi.md]:
```suggestion
sidebar_label: 'Wagmi Web3js Adapter'
```

File: docs/docs/guides/wagmi_usage/wagmi.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+title: 'Wagmi Web3js Adaptor'
```
Comment (jdevcs) [inline docs/docs/guides/wagmi_usage/wagmi.md]:
```suggestion
title: 'Wagmi Web3js Adapter'
```

**Comment by avkos** [comment]
Here is the wagmi docs PR
https://github.com/wevm/wagmi/pull/3624

**Review by luu-alex — APPROVED** [review]

**Pull request merged**